R/deployApp.R

```diff
@@ -288,23 +292,26 @@ deployApp <- function(appDir = getwd(),
     python <- getPythonForTarget(python, accountDetails)
     pythonConfig <- pythonConfigurator(python, forceGeneratePythonEnvironment)
 
+    if (!quiet) {
+      cli::cli_alert_info("Bundling files: {.file {appFiles}}")
+    }
+    bundlePath <- bundleApp(
+      appName = target$appName,
+      appDir = appDir,
+      appFiles = appFiles,
+      appPrimaryDoc = appPrimaryDoc,
+      contentCategory = contentCategory,
+      verbose = verbose,
+      pythonConfig = pythonConfig,
+      quarto = quarto,
+      isCloudServer = isCloudServer(accountDetails$server),
+      metadata = metadata,
+      image = image
+    )
+
     # create, and upload the bundle
     logger("Bundle upload started")
-    withStatus(paste0("Uploading bundle (", application$id, ")"), {
-      bundlePath <- bundleApp(
-        appName = target$appName,
-        appDir = appDir,
-        appFiles = appFiles,
-        appPrimaryDoc = appPrimaryDoc,
-        contentCategory = contentCategory,
-        verbose = verbose,
-        pythonConfig = pythonConfig,
-        quarto = quarto,
-        isCloudServer = isCloudServer(accountDetails$server),
-        metadata = metadata,
-        image = image
-      )
-
+    withStatus("Uploading bundle", {
```

Comment:
Is it silly to have `logger` immediately followed by `withStatus`, which also logs?